workflowproj/operator.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflowproj
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kiegroup/kogito-serverless-operator/api/metadata"
+	operatorapi "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
+)
+
+const (
+	workflowConfigMapNameSuffix = "-props"
+	// ApplicationPropertiesFileName is the default application properties file name
+	ApplicationPropertiesFileName = "application.properties"
+	// LabelApp key to use among object selectors
+	LabelApp = "app"
```

Comment:
maybe kogito-app or swf-app?

Reply:
This can't be any different since it's the globally accepted label key for applications deployed on OpenShift/k8s. IIRC they use it to display the object's relation in the console.